lib/stanford/storage_object_validator.rb

```diff
@@ -11,7 +11,8 @@ class StorageObjectValidator < Moab::StorageObjectValidator
     def validation_errors
       errors = []
       errors.concat super
-      errors.concat identify_druid
+      errors.concat(identify_druid) if errors.empty?
+      errors
```

Comment:
I do not care if this is addressed now (or ever) but ... do we need L15?

Reply:
IIRC, line 14 will return `nil` if the if condition isn't met...

yeah:
```ruby
[1] pry(main)> 'hello' if true
=> "hello"
[2] pry(main)> 'hello' if false
=> nil
[3] pry(main)> 
```